packages/api/src/schema/schema.yaml

```diff
@@ -2572,6 +2572,70 @@ components:
           type: number
           example: 200000
 
+    create-room-response:
+      additionalProperties: false
+      properties:
+        id:
+          type: string
+
+    room:
+      type: object
+      table: rooms
+      required:
+        - id
+        - participants
+        - events
+      properties:
+        id:
+          type: string
+          unique: true
+          index: true
+          readOnly: true
+        userId:
+          type: string
+          readOnly: true
+          example: 66E2161C-7670-4D05-B71D-DA2D6979556F
+          index: true
```

Comment:
You need to add `room` to this list here: https://github.com/livepeer/livepeer-com/blob/a326c2f31bfd7638390ba703bcf7f1ca6a7cad1e/packages/api/src/store/table.ts#L352

Otherwise the indexes won't really be created. This is a tech debt we've been carrying since the time we had some issues with indexes on the `stream` table which is huge and gets a lot of updates.